features/datum.feature

```diff
@@ -310,106 +308,99 @@ Functionaliteit: leveren van een datum
+        Dan heeft de response een persoon met een '<danrelatie>' met de volgende '<groep>' gegevens
+        | naam                | waarde        |
+        | burgerservicenummer | true          |
+        | <veld>.type         | <type>        |
+        | <veld>.datum        | <datum>       |
+        | <veld>.jaar         | <jaar>        |
+        | <veld>.maand        | <maand>       |
+        | <veld>.onbekend     | <onbekend>    |
+        | <veld>.langFormaat  | <langFormaat> |
 
       Voorbeelden:
-      | relatie | field    | groep       | veld                 | type           | GBA InOnderzoek waarde | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | Datum          | 020000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 020000                 | 00000000 |            |      |       | true     | onbekend     |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 020000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 020000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | Datum          | 050000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 050000                 | 00000000 |            |      |       | true     | onbekend     |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 050000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 050000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | Datum          | 090000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 090000                 | 00000000 |            |      |       | true     | onbekend     |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 090000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 090000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
+      | relatie   | danrelatie | field       | groep                | veld                 | type           | GBA InOnderzoek waarde | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
```

Comment:
De kolommen 4 en 5 van deze tabel kunnen verwijderd worden. De waardes kunnen hard in het scenario geplaatst worden aangezien ze voor alle voorbeelden gelijk zijn.

Reply:
Aangepast